features/step_definitions/package_json_steps.coffee

```diff
@@ -4,11 +4,13 @@ path = require 'path'
 
 module.exports = ->
 
-  @Given /^I am in a node project$/, (done) ->
-    fsExtra.outputJson path.join(@tmpDir, 'package.json'), name: '123', done
+  @Given /^I work on a Node.js project$/, (done) ->
```

Comment:
I like suggestion (1) and have adopted it. Regarding suggestion (2), Node.js applications also have a `package.json` file. Also, Mycha is for both Node.js apps and packages, so I prefer the more generalized term `project` here.